src/main/tables/107_default_instant_launches.sql

```diff
@@ -0,0 +1,13 @@
+SET search_path = public, pg_catalog;
+
+CREATE TABLE IF NOT EXISTS default_instant_launches (
+    id UUID UNIQUE NOT NULL DEFAULT uuid_generate_v1(),
+
+    version SERIAL NOT NULL,
```

Comment:
I'm not sure if I'm missing something, but it seems like these columns duplicate each other. The version here being set to `SERIAL` suggests to me that this field will be globally unique, rather than being a version within some given set. Otherwise, we'd need to be manually inserting things that ignore the sequence (i.e., don't use the default value), but I'm not sure what that would be useful for.

Basically, I'm curious if there's a reason to have the `id` column rather than using `version` as the primary key.

As a side note, `SERIAL` already implies `NOT NULL`, re [docs](https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-SERIAL) -- basically it works out to `integer NOT NULL DEFAULT nextval('the_sequence_created_automatically_by_postgres_for_this')`

Reply:
I added a comment to the file to this effect, but the intention is that we can maintain a history of edits to the default instant launches so that we can make edits safely without having to parse or modify primary keys in the database.

Reply:
Actually, my comment is incorrect. Fixing now. Sorry, caffeine hasn't kicked in fully yet.

Reply:
Okay, now it's fixed.